README.md

```diff
@@ -69,25 +69,28 @@ Everything is organised:
 
 ### Installation on YunoHost
 
-- Firefly III requires a **dedicated (sub)domain** (must install under web root like **https://firefly.example.com/** not **https://example.com/firefly/** )
-
-#### Register a new domain and add it to YunoHost
-
-Firefly III requires a dedicated (sub)domain, so obtain one and add it using the YunoHost admin panel. **Domains -> Add domain**. As Firefly III uses the full domain and is installed on the root, you can create a subdomain such as firefly.domain.tld. Don't forget to update your DNS if you manage them manually.
+- Tested under a subdirectory on an existing domain
+- Tested on amd64 and arm64 arches
 
 #### Install the Firefly III application
 
 Use the YunoHost admin panel to install Firefly III by entering the GitHub repo address in the custom app URL:
 
 ```
-https://github.com/YunoHost-Apps/firefly-iii_ynh
+https://github.com/dr41nU/Firefly-III_ynh
```

Comment:
What’s the reason for the changed github link?

Reply:
Hello,
Cause formerly I didn't aim creating a pull request
Sorry for the inconvenience, I will try to revert this change
Regards,
Laurent

Reply:
I think it should be OK now
Thanks